### PR TITLE
Add tests in conjunction with fix for issue #1045 in cadasta-platform

### PR DESCRIPTION
**Pull request by shailysangwan**
As suggested in https://github.com/Cadasta/cadasta-project-ideas/issues/17, I have added the tests as part of the fix for https://github.com/Cadasta/cadasta-platform/issues/1045, to go along with PR https://github.com/Cadasta/cadasta-platform/pull/1134.

### CODE REVIEW

File: selenium_tests/accounts/registration_form_validation.py [hunk 1]
```diff
@@ -139,4 +139,48 @@ def test_empty_confirm_password(self):
...
+        self.wd.find_css('#id_username').send_keys('')
+        self.wd.find_css('#id_email').send_keys("user2@abc.com")
+        self.wd.find_css("#id_password1").send_keys('XYZ#qwerty')
+        self.wd.find_css("#id_password2").send_keys('XYZ#qwerty')
+        self.wd.find_css("#id_full_name").send_keys('user2-name')
+        action = ActionChains(self.wd)
+        action.send_keys(Keys.TAB).send_keys(Keys.RETURN).perform()
+
+        text = self.wd.find_elements_by_xpath("//ul[contains(@class, 'parsley-errors-list')]")
+        assert len(text) == 1
```
Comment (shailysangwan) [inline selenium_tests/accounts/registration_form_validation.py]: Expecting only one error after the fix

**Comment by oliverroick** [comment]
@manoramahp Would you be able to review this PR and merge it in if you think it's good to go? You're closest to the code base at the moment so I think you the best person for this.

**Comment by manoramahp** [comment]
@shailysangwan, @oliverroick This PR looks good. Hence merged.

**Comment by manoramahp** [comment]
@shailysangwan Thank you for this PR. You can add these two test cases here in the cadasta-test wiki, so that any contributor get to know what are the already implemented test cases.
https://github.com/Cadasta/cadasta-test/wiki/User-Accounts-Test-Cases

**Comment by shailysangwan** [comment]
@manoramahp I had tried doing that, but I guess I don't have the [permissions](https://help.github.com/articles/changing-access-permissions-for-wikis/).

**Pull request merged**